server/text_generation_server/utils/hub.py

```diff
@@ -26,7 +26,10 @@ def weight_hub_files(
     filenames = [
         s.rfilename
         for s in info.siblings
-        if s.rfilename.endswith(extension) and len(s.rfilename.split("/")) == 1
+        if s.rfilename.endswith(extension)
+        and len(s.rfilename.split("/")) == 1
+        and "arguments" not in s.rfilename
+        and "args" not in s.rfilename
```

Comment:
Perhaps it's good to have a dummy repo on the Hub (e.g. under `hf-internal-testing`) that we can use to unit test this?